.buildkite/pipelines/es_serverless/emergency_relelease_branch_testing.yml

```diff
@@ -0,0 +1,22 @@
+# https://buildkite.com/elastic/kibana-serverless-emergency-release-branch-testing
+
+## Runs regular PR testing and builds the artifacts container image for emergency releases
+
+notify:
+  - slack: "#kibana-mission-control"
+    if: "build.state == 'passed' || build.state == 'failed' || build.state == 'scheduled'"
+
+steps:
+  - trigger: "kibana-pull-request"
```

Comment:
https://buildkite.com/elastic/kibana-pull-request/builds/174247#018baf4c-e669-4c0d-b202-b0b9d1e973f4/140-142

This will probably have to be kibana-on-merge.

Reply:
@jbudz Thanks for the check, I didn't think about trying it out up-front.

According to the original issue, we wanted the on-merge, but I had the concern, that it might have parts to it that are meant for commits that landed in `main` (e.g.: creating and uploading artifacts from main snapshots).

Now I see that this job, on the other hand, has the issue that there's no PR connected to it.

To me, neither seems quite OK for this case. Could I instead upload the PR job's `base.yml` (and the `post-build.yml`)?

Reply:
Maybe I misunderstood something, but is this step required for triggering `kibana-artifacts-container-image` below?

As described in https://github.com/elastic/kibana/issues/170828#issuecomment-1801915105, for now we just want to build the container images and we trust that the successful PR CI run is enough to validate the commit (the one that runs prior to a commit being merged into the `deploy-fix@` branch). So there shouldn't be a need for running the CI again in this pipeline as far as I see it.

Reply:
I only read the update comment on that thread now, that makes this test section unnecessary.

With this, however, I feel that setting up a pipeline only to trigger an automatic build for that one pipeline (with a different notification channel) might be a bit of an overkill, wouldn't it? We could just add an extra step to the emergency-release playbook to generate an artifact build from the branch when it's deemed ready.

Reply:
I removed the testing bit based on the issue comment: https://github.com/elastic/kibana/issues/170828#issuecomment-1801915105